guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,21 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using LUKS during host provisioning.
+In addition, you can bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign it to your host group or select it manually during provisioning.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
++
+* If you want to use Linux Unified Key Setup (LUKS) to encrypt and decrypt the disks of your host, add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
+If a value for `disk_enc_passphrase` is not specified, the default value is `linux`.
+* If you want to use Clevis/Tang to decrypt the disk of your hosts during boot time, add the host parameter `disk_enc_tang_servers` as type *array* and your list of Tang servers (example: `["1.2.3.4"]` or `["{server-example-com}", "5.6.7.8"]`).
++
+You do not have to set `disk_enc_passphrase` because the passphrase slot is removed from the LUKS container after provisioning.
```

Comment:
Does that mean there is no passphrase left to unlock the LUKS-container after successful provisioning?
Would this not mean that before rebooting the machine, one has to set a new passphrase to the LUKS-container and configure it to allow the system to unlock it during boot (or at all)?